schema/transportation/segment.yaml

```diff
@@ -128,166 +140,28 @@ properties:
         motor_vehicle includes car, truck and motorcycle
     road:
       description: >-
+        ** Note: this field is deprecated in favor of promoting its sub-properties into top-level properties and will be removed in the 2024-07 release **
+
         Properties for segments whose segment subtype is road. The road
         subtype includes any variety of road, street, or path, including
         dedicated paths for walking and cycling.
       type: object
       unevaluatedProperties: false
```

Comment:
Since the actual properties are deleted, does it make sense to explicitly set `unevaluatedProperties: true` until the `road` property is deleted? Or alternatively, just to not delete the sub-properties?

Reply:
I didn't delete the sub-properties, just shuffled them into containers for reuse. Did I miss something?

Reply:
Nope, it seems like I missed something!

Reply:
Yeah I definitely read the red and not the green. That's a pretty rookie mistake. Sorry for the churn, but I'll resolve this one.